rts/lra/service-base/src/main/java/io/narayana/lra/logging/LraI18nLogger.java

```diff
@@ -126,21 +113,17 @@
     @Message(id = 25020, value = "Cannot process non JAX-RS LRA participant")
     void error_cannotProcessParticipant(@Cause ReflectiveOperationException e);
 
-    @LogMessage(level = ERROR)
     @Message(id = 25021, value = "Invalid format of LRA id to be converted to LRA coordinator url, was '%s'")
-    void error_invalidLraIdFormatToConvertToCoordinatorUrl(String lraId, @Cause Throwable t);
+    String error_invalidLraIdFormatToConvertToCoordinatorUrl(String lraId, @Cause Throwable t);
 
-    @LogMessage(level = INFO)
     @Message(id = 25022, value = "Failed enlisting to LRA '%s', coordinator '%s' responded with status '%d (%s)'. Returning '%d (%s)'.")
-    void info_failedToEnlistingLRANotFound(URL lraId, URI coordinatorUri, int coordinatorStatusCode,
-            String coordinatorStatusMsg, int returnStatusCode, String returnStatusMsg);
-
```

Comment:
We don't allow formatting changes amongst functional changes (ie a blank line has been removed